lib/packwerk/parsers/erb.rb

```diff
@@ -19,6 +19,10 @@ def initialize(parser_class: BetterHtml::Parser, ruby_parser: Ruby.new)
       def call(io:, file_path: "<unknown>")
         buffer = Parser::Source::Buffer.new(file_path)
         buffer.source = io.read
+        parse_buffer(buffer, file_path: file_path)
+      end
+
+      def parse_buffer(buffer, file_path:)
```

Comment:
I split this to allow subclasses to implement `#parse_buffer` without concerning themselves with the actual file read.